source/core/src/main/com/csse3200/game/ui/DialogueBox/DialogBox.java

```diff
@@ -0,0 +1,77 @@
+package com.csse3200.game.ui.DialogueBox;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
```

Comment:
Same as below - remove redundant imports and is this meant to be a part of this PR?